workflow.md

```diff
@@ -54,6 +54,12 @@ project manager would do it himself). Do not merge your own PRs!
 Please describe in PR what have you done, if it is tested or not, any additional
 information.
 
+## Merging Pull requests
+
+After review by another team member, the teammate merging the PR should merge
+`master` into the Pull request's branch *without* the `--no-ff` option, and then
```

Comment:
@gromakovsky Thanks for clarifying. So, as long as it doesn't turn out to be too complicated, I can rebase and keep the history clean, but I shouldn't waste too much time on it if a merge would be significantly simpler to do?